ccc/parameters.py

```diff
@@ -3,37 +3,39 @@
 import six
 import re
 import numpy as np
-
-import taxcalc
+import paramtools
 
 # import ccc
 from ccc.get_taxcalc_rates import get_rates
 from ccc.utils import DEFAULT_START_YEAR
+CURRENT_PATH = os.path.abspath(os.path.dirname(__file__))
 
-
-class Specification(taxcalc.Parameters):
+class Specification(paramtools.Parameters):
     '''
-    Inherits Tax-Calculator Parameters abstract base class.
+    Inherits ParamTools Parameters abstract base class.
     '''
-
-    DEFAULTS_FILE_NAME = 'default_parameters.json'
-    DEFAULTS_FILE_PATH = os.path.abspath(os.path.dirname(__file__))
-    DEFAULTS_FIRST_YEAR = 2015
-    DEFAULTS_LAST_YEAR = 2028
-    DEFAULTS_NUM_YEARS = DEFAULTS_LAST_YEAR - DEFAULTS_FIRST_YEAR + 1
+    defaults = os.path.join(CURRENT_PATH, "default_parameters.json")
+    # array_first = True
```

Comment:
@hdoupe I'm trying to bring `paramtools.parameters()` back into the `ccc.Specifcation()` object.  I'm having some issue with formatting parameters as lists/arrays. I'm using ParamTools 0.6.0.  If I do `array_first = True' I get an error:
```
f"The Value objects for {param} do not span the specified "
paramtools.exceptions.SparseValueObjectsException: The Value objects for CIT_rate do not span the specified parameter space. Missing combinations:
	(2019,)
	(2020,)
	(2021,)
	(2022,)
	(2023,)
	(2024,)
	(2025,)
	(2026,)
	(2027,)
	(2028,)
```
 If I comment out this line, then I have issues where parameters are lists when arrays are expected:

```
File "/Users/jason.debacker/repos/CCC/ccc/parameters.py", line 84, in compute_default_params
    sprime_c_td = ((1 / self.Y_td) *
TypeError: unsupported operand type(s) for /: 'int' and 'list'
```

Can you advise on how I can get this to work correctly?

Reply:
@jdebacker The array extension is off by default. If you set the `label_to_extend` class attribute to `"year"`, then it should work. Here's an [example](https://paramtools.org/api/extend/) in the docs.